uvm/impl/com/untangle/uvm/EventManagerImpl.java

```diff
@@ -502,6 +499,14 @@ private LinkedList<AlertRule> defaultAlertRules()
         EventRuleCondition condition3;
         AlertRule eventRule;
 
+        conditions = new LinkedList<>();
+        condition1 = new EventRuleCondition( "class", "=", "*CriticalAlertEvent*" );
+        conditions.add( condition1 );
+        condition2 = new EventRuleCondition( "component", "=", "DISK_CHECK_FAILURE" );
+        conditions.add( condition2 );
+        eventRule = new AlertRule( true, conditions, true, true, "Disk health checks failed", false, 0 );
+        rules.add( eventRule );
+
         conditions = new LinkedList<>();
```

Comment:
How about clearing already created conditions object on line ##502 and reuse that instead of creating a new object here ?